content/docs/15.how-to-guides/sqlmesh.md

```diff
@@ -0,0 +1,195 @@
+5. Create SQLMeshCLI task that will run the dbt project.
+
+### How will we implement dbt project?
+
+SQLMesh supports integration with a variety of tools like Airflow, dbt, dlt, etc. One of the common use-cases of SQLMesh is to run dbt projects.
+
+You can choose to pull your dbt project from Git repository as mentioned in the [how-to-guide on dbt](docs/how-to-guides/dbt) or create the namespace files for the complete project. Here, we will create the complete project using namespace files which we will create on the fly. You can later choose to push all the namespace files to GitHub repository using [PushNamespaceFiles](docs/how-to-guides/pushnamespacefiles).
+
+### Creating Flow with SQLMeshCLI task
+
+We will first create the Kestra flow.
+
+Based on the steps mentioned in the description, lets create tasks for each step.
+
+```yaml
+id: sqlmesh_transform
+namespace: company.team
+
+tasks:
+  - id: orders_http_download
+    type: io.kestra.plugin.core.http.Download
+    description: Download orders.csv using HTTP Download
+    uri: https://huggingface.co/datasets/kestra/datasets/raw/#main/csv/orders.csv
+
+  - id: create_orders_table
+    type: io.kestra.plugin.gcp.bigquery.CreateTable
+    description: Create orders table in BigQuery
+    serviceAccount: "{{ secret('GCP_SERVICE_ACCOUNT_JSON') }}"
+    projectId: <gcp-project-id>
+    dataset: ecommerce
+    table: orders
+    tableDefinition:
+      type: TABLE
+      schema:
+        fields:
+        - name: order_id
+          type: INT64
+        - name: customer_name
+          type: STRING
+        - name: customer_email
+          type: STRING
+        - name: product_id
+          type: INT64
+        - name: price
+          type: FLOAT64
+        - name: quantity
+          type: INT64
+        - name: total
+          type: FLOAT64
+
+  - id: load_orders_table
+    type: io.kestra.plugin.gcp.bigquery.Load
+    description: Load orders table with data from orders.csv
+    from: "{{ outputs.orders_http_download.uri }}"
+    projectId: <gcp-project-id>
+    serviceAccount: "{{ secret('GCP_SERVICE_ACCOUNT_JSON') }}"
+    destinationTable: "<gcp-project-id>.ecommerce.orders"
+    format: CSV
+    csvOptions:
+      fieldDelimiter: ","
+      skipLeadingRows: 1
+
+  - id: sqlmesh_transform
+    type: io.kestra.plugin.sqlmesh.cli.SQLMeshCLI
+    description: Use SQLMesh to run the dbt project
+    inputFiles:
+      sa.json: "{{ secret('GCP_SERVICE_ACCOUNT_JSON') }}"
+    namespaceFiles:
+      enabled : true
+    beforeCommands:
+      - pip install "sqlmesh[bigquery]"
+      - pip install dbt-bigquery
+    commands:
+      - sqlmesh init -t dbt
+      - sqlmesh plan --auto-apply
+```
+
+Ensure you set the `namespaceFiles` property with `enabled: true` to ensure that the task has access to namespaceFiles. Provide the GCP service account JSON file so that the task is able to connect to GCP account in order to access BigQuery. This file is referenced in the dbt project file.
```

Comment:
```suggestion
Ensure you set the `namespaceFiles` property with `enabled: true` set to ensure that the task has access to namespace files. Also, provide the GCP service account JSON file so that the task is able to connect to your GCP account in order to access BigQuery. This file is referenced in the dbt project file.
```